BugDoom/src/main/java/com/infoshareacademy/service/DataHandler.java

```diff
@@ -0,0 +1,98 @@
+package com.infoshareacademy.service;
+
+import com.google.gson.Gson;
+import com.infoshareacademy.model.*;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class DataHandler {
+    public static <T> void appendToFile(T objectToSave) {
+        // saving an object to Json
+        Gson gson = new Gson();
```

Comment:
```suggestion
public class DataHandler {
    Gson gson = new Gson();
    public static <T> void appendToFile(T objectToSave) {
```
Gson nie będzie tworzony za każdym wywołaniem metody co pozwoli nam zaoszczędzić trochę zasobów